### PR TITLE
Adding new file about developing locally and running in Kubernetes

### PR DESCRIPTION
**Pull request by sishida**
Signed-off-by: Sarah Ishida <sishida@us.ibm.com>

Note: This PR is not yet ready to merge.

This PR is part of https://github.com/eclipse/codewind/issues/393.

### CODE REVIEW

File: docs/_documentations/mdt-vsc-hybridscenario.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+   - Enter your user name and password.
+7. Save your information. If your info is correct, the page refreshes, your user name appears without the password, and a connection appears.
+
+**Note:** If the info for either the **Deployment** or **Docker Registry** areas is incorrect, an error appears in the sidebar, and you can try again.
+- Codewind cannot connect to a deployment or a server. Edit or open the remote connection.
+- No projects exist. Create a project.
+
+8. After you create a connection, if you need to suspend or remove it, you can disconnect or delete it.
+    - Disconnect:
+    - Delete:
```
Comment (sishida) [inline docs/_documentations/mdt-vsc-hybridscenario.md]: We need to explain what each of these options does and the difference between them.

File: docs/_documentations/mdt-vsc-hybridscenario.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+order:
+parent:
+---
+# Developing applications locally and building and running Kubernetes
+Develop your applications on your local machine and then use Kubernetes to build and run them on the cloud.
+
+1. Click the **Add remote connection** icon. The **Add remote connection** window appears.
+2. Enter a name for your connection and click **Enter** to create the remote connection. The remote connection page appears.
+3. Two peices are required to create a remote connection: where your project is being created and where you want to connect to the cloud.
+   - **Deployment** is the location in which you are creating your project.
```
Comment (sishida) [inline docs/_documentations/mdt-vsc-hybridscenario.md]: "Deployment" needs a better name.

File: docs/_documentations/mdt-vsc-hybridscenario.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+   - **Docker registry** is where you need a connection to Docker to build and run your project.
+4. Fill in the following information in the **Deployment** area:  
+   - Enter the **URL** that you need to log in to your cloud.
+   - Enter your **Realm**. Your administrator provides this information.
+   - Enter your **ClientID**. Your administrator provides this information.
```
Comment (sishida) [inline docs/_documentations/mdt-vsc-hybridscenario.md]: We could use more information about these fields, such as where to find them.

File: docs/_documentations/mdt-vsc-hybridscenario.md (hunk 2 above)
Comment (jgwest) [reply docs/_documentations/mdt-vsc-hybridscenario.md]: `peices` 👀

Comment (sishida) [reply docs/_documentations/mdt-vsc-hybridscenario.md]: Good catch! I'll fix that.

**Comment by sishida** [comment]
This file is no longer needed. I am going to close this PR.

**Pull request closed without merging**